probe/core/src/main/java/org/jboss/weld/probe/InvocationMonitor.java

```diff
@@ -86,7 +86,7 @@
 
     @Intercepted
     @Inject
-    private Bean<?> interceptedBean;
+    private transient Bean<?> interceptedBean;
```

Comment:
It is injected by the container and the spec requires the beans to be passivation capable deps, see also http://docs.jboss.org/cdi/spec/1.2/cdi-spec.html#bean_metadata and `org.jboss.weld.bean.builtin.BeanMetadataBean.newInstance(InjectionPoint, CreationalContext<Bean<?>>)`.